src/components/shared/ChatListItem.js

```diff
@@ -125,7 +131,10 @@ class Shared extends Component<Props, State> {
                 : <Icon5 name="meh" size={40} color={colors.dusk} light/>
             }
           </View>
-          <Text style={styles.txtPeerMsg}>{this.props.item.message}</Text>
+          <View style={styles.wrapperPeerMsg}>
+            <Text style={styles.txtPeerName}>{this.props.item.sender}</Text>
```

Comment:
It's so nice. :+1:

But, I think it will show images and sender names in each message when the same user sends two or more messages.
If you want images and names to appear once, you should make this component different.

I think it's a good way to register in issues and to work on it on another branch.